network/src/lib.rs

```diff
@@ -165,10 +197,10 @@ pub enum Message {
 	RequestPovBlock(RequestId, Hash, Hash),
 	/// Provide requested proof-of-validation block data by candidate hash or nothing if unknown.
 	PovBlock(RequestId, Option<PoVBlock>),
-	/// Request block data (relay_parent, candidate_hash)
-	RequestBlockData(RequestId, Hash, Hash),
+	/// Request block chunk (relay_parent, candidate_hash, chunk id)
+	RequestBlockChunk(RequestId, Hash, Hash, u64),
```

Comment:
Could we declare and then import as necessary some type aliases like the following so the code is readable without extra code comments?
```
pub type RelayParent = T::Hash;
pub type CandidateHash = T::Hash;
pub type ChunkId = u64;
```

Then change this line to:

```suggestion
	RequestBlockChunk(RequestId, RelayParent, CandidateHash, ChunkId),
```

Reply:
I think so, but we also need to do that for other variants of this enum like `RequestPovBlock`, and it would be better to do that in a separate changeset.